include/geolib/ros/msg_conversions.h

```diff
@@ -3,8 +3,11 @@
 
 #include "geolib/datatypes.h"
 #include <geometry_msgs/Pose.h>
+#include <geometry_msgs/Point.h>
 #include <geometry_msgs/Point32.h>
 #include <geometry_msgs/Transform.h>
+#include <geolib/Mesh.h>
```

Comment:
Suggestion, move this line near the other geolib includes?

I see we make a mess of lowercase and uppercase filenames too, how nice :p